wordpress-mobile/WordPress-iOS
# Activity Log: Rewind

**Another step for** #7832

Done on top of #8225 and not to be merged until that one is.

**To test:**

1. Go to the activity log and find a rewindable activity (blue ribbon on the left).
2. Tap on it and select Rewind.
3. Wait for it.
4. Wait for it.
5. Wait for it.
6. Check on the web that the site was properly rewound.
7. Leave the screen and come back and check on the phone that the site was probably rewound.

Notes:
As with this series of PRs this is the simplest UI to make the feature work, not intended for actual use by human beings.
